igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -0,0 +1,108 @@
+#include <ros/ros.h>
+#include <nav_msgs/Odometry.h>
+#include <mutex>
+#include <geometry_msgs/Vector3.h>
+#include <tf/transform_broadcaster.h>
+#include <tf/transform_listener.h>
+#include <tf/transform_datatypes.h>
+#include <igvc_utils/NodeUtils.hpp>
+#include <robot_localization/navsat_conversions.h>
+
+ros::Publisher ground_truth_pub;
+std::mutex mutex;
+nav_msgs::Odometry og_pose;
+ros::Time last_estimate;
```

Comment:
globals should almost never be used. When used, they should be prefixed with g_.
http://wiki.ros.org/CppStyleGuide#Global_variables